sdk/src/Services/S3/Custom/Transfer/BaseDownloadRequest.cs

```diff
@@ -271,5 +272,15 @@ public ChecksumMode ChecksumMode
             get { return this.checksumMode; }
             set { this.checksumMode = value; }
         }
+
+        /// <summary>
+        /// <para>Confirms that the requester knows that they will be charged for the request. 
```

Comment:
Is `<para>` required here? It's not used in `GetObjectRequest`: https://github.com/aws/aws-sdk-net/blob/875eb2d705bb0cdd8b8b602780494e7b8e0f13d3/sdk/src/Services/S3/Custom/Model/GetObjectRequest.cs#L468-L476

The `.` is also after the tag ends, it ends up looking like this in Intellisense:

![intellisense](https://github.com/aws/aws-sdk-net/assets/3842788/e056fa3b-a632-49c1-8633-abb4449affea)

Reply:
Thanks for the feedback. Fixed.